src/algorithms/reco/FarForwardNeutronReconstruction.cc

```diff
@@ -7,20 +7,25 @@
 #include <edm4hep/Vector3f.h>
 #include <math.h>
 #include <gsl/pointers>
+#include <vector>
 
 /**
- Creates a "neutron candidate" Reconstructed Particle consisting of all clusters in a given ClusterCollection.
- Its energy is the sum of the energies of the constituent clusters, and its direction is the direction from the
-origin to the position of the most energetic cluster
+ Creates a "neutron candidate" Reconstructed Particle consisting of all clusters in a
+ given ClusterCollection.  Its energy is the sum of the energies of the constituent clusters
+ times a correction factor, and its direction is the direction from the origin to the position
+ of the most energetic cluster.  The correction factor is given by 1/(1+c[0]+c[1]/sqrt(E)+c[2]/E),
```

Comment:
Is there a reference for this form of correction? If yes, can we add a link to this comment?

Reply:
Hi @veprbl ,
I decided on this form of a correction as an "empirical fit" to the distributions of the discrepancy between the uncorrected reconstructed energies and the truth energies in the MC simulations in some studies I have done.  I did not derive this from first principles, nor have I seen this formula somewhere else that I can refer back to.  How I got this was I noticed that at lower energies, the discrepancies were larger and at higher energies they leveled off, and were approximately flat.  I first tried fitting to a+c/E, but this clearly didn't fit very well.  Then I tried a+b/sqrt(E)+c/E, and this worked very well.  
Best regards,
Sebouh

Reply:
Understood. Thanks for updating the comment.